awc/src/test.rs

```diff
@@ -58,6 +58,19 @@ impl TestResponse {
         panic!("Can not set header");
     }
 
+    // TODO: Should `set` be removed and replaced with this instead?
```

Comment:
Yes, please. Looking below, also header => append_header, with the same signature as this.